fe1-web/src/features/wallet/screens/WalletError.tsx

```diff
@@ -16,22 +15,18 @@ const styles = StyleSheet.create({
 /**
  * Wallet synchronization error screen
  */
-const WalletError = ({ navigation }: IPropTypes) => (
-  <View style={containerStyles.centeredY}>
-    <TextBlock text={STRINGS.wallet_error} />
-    <View style={styles.largePadding} />
-    <WideButtonView
-      title={STRINGS.back_to_wallet_home}
-      onPress={() => navigation.navigate(STRINGS.navigation_home_tab_wallet)}
-    />
-  </View>
-);
-
-const propTypes = {
-  navigation: PROPS_TYPE.navigation.isRequired,
+const WalletError = () => {
+  const navigation = useNavigation<any>();
```

Comment:
You should maybe add a `FIXME` comment here, as it's not intended to use `any` and it should be fixed in the future.

Reply:
I will add the navigation type on another PR I think, this PR is meant to be a small fix for the navigation system.